dallinger/frontend/templates/base/dashboard.html

```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <title>{% block title %}{{title}} - {{app_id}} Dashboard{% endblock %}</title>
         {% block replace_stylesheets %}
-          <link rel="stylesheet" href="/static/css/bootstrap.min.css" type="text/css">
+          <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```

Comment:
Why is this being moved to a CDN? This will prevent users from changing the bootstrap version (which may or may not be good).

Reply:
I don't actually want to move this to a CDN, but the bootstrap version currently just comes along for the ride with the demo or your custom experiment, and in the case of the demos it's a pretty ancient version. There are some basic nav layouts available with newer versions I wanted to try, so I grabbed the CDN for expediency's sake. 

I do think there's an decision to make here: do we want all the dashboard stuff to be at the mercy of the custom experiment's CSS inclusions, or do we want it to pull in its own?